cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
@hferentschik
```
$ minishift hostfolder add --user-home'?
```

I agree to push it into a flag, just not sure about `--user-home`, as it is actually is the whole `Users` folder as a share... not specifically that specific user's folder (while it being part of it). maybe `--users-share` would be more appropriate.